.github/workflows/ci.yml

```diff
@@ -209,6 +212,22 @@ jobs:
           if [[ $GITHUB_REPOSITORY =~ /(silverstripe-spellcheck|recipe-authoring-tools)$ ]] || [[ "${{ matrix.phpunit_suite }}" == "recipe-authoring-tools" ]]; then
             sudo apt install -y hunspell libhunspell-dev hunspell-en-us
           fi
+          if [[ $GITHUB_REPOSITORY =~ /(silverstripe-dynamodb)$ ]]; then
+            # install dynamodb
+            mkdir _dynamodb
+            cd _dynamodb
+            sudo wget https://s3.ap-southeast-1.amazonaws.com/dynamodb-local-singapore/dynamodb_local_latest.tar.gz
+            sudo tar -xf dynamodb_local_latest.tar.gz
+            # run dynamo db as background process
+            # github actions comes java - `java -version` = openjdk version "11.0.18" 2023-01-17
+            $(java -Djava.library.path=./DynamoDBLocal_lib -jar DynamoDBLocal.jar) &
+            sleep 3
+            if ! [[ $(ps -aux | grep DynamoDBLocal.jar) =~ DynamoDBLocal\.jar ]]; then
```

Comment:
What is the `sleep 3` there for?

Reply:
Ensure the that DynamoDBLocal.jar has started, it's running as a background process so it goes to the next line without waiting for the command to finish.  In hindsight I should change this to a for loop and check to that it's started